main.css

```diff
@@ -7,6 +7,79 @@ html, body
     flex-direction: column;
 }
 
+footer
+{
+    position: fixed;
+    bottom: 0;
+    right: 0;
+    width: 100%;
+    height: 20px;
+    z-index: 9999;
+    background-color: transparent;
+}
+
+#notifications
+{
+    position: fixed;
+    top: 45px;
+    right: 0px;
```

Comment:
We're going to have another UI component in the top right, so we actually want these notifications to span right across the center, between the two. I think if change this from `right` to `left`, and give it `300px` we can clear the sidebar and have enough room for the new UI on the desktop, but we should also make sure we have a media query, because it should span the _whole_ top on mobile (`left: 0`)